src/network/networkctl.c

```diff
@@ -1473,10 +1478,29 @@ static int link_status_one(
                                    TABLE_BOOLEAN, info->stp_state > 0,
                                    TABLE_EMPTY,
                                    TABLE_STRING, "Multicast IGMP Version:",
-                                   TABLE_UINT8, info->mcast_igmp_version);
+                                   TABLE_UINT8, info->mcast_igmp_version,
+                                   TABLE_EMPTY,
+                                   TABLE_STRING, "Cost:",
+                                   TABLE_UINT32, info->cost);
                 if (r < 0)
                         return table_log_add_error(r);
 
+                if (info->port_state <= BR_STATE_BLOCKING) {
+                        static const struct {
+                                const char *state;
+                        } state_table[] = {
+                                { "disabled" },
+                                { "listening" },
+                                { "learning" },
+                                { "forwarding" },
+                                { "blocking" },
```

Comment:
I can't say I like this. A separate table with symbolic names for the indices would be nicer. But it *will* work correctly, so I guess it's OK to merge. If we ever need to do this translation in a different context, we can move this out to and create normal _to_string()/_from_string() helpers.